docs: add proposed project ideas to mentorship directory (#1609)
asyncapi/community: mentorship/asyncapi-mentorship/2024/project-ideas.md

8 | [Integrating EDAVisualiser Application Focus View into VS Code AsyncAPI Plugin](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10540858) | [AsyncAPI Preview](https://github.com/asyncapi/vs-asyncapi-preview) | [Ivan](https://github.com/ivangsa) |Enhance the VS Code AsyncAPI ([asyncapi/vs-asyncapi-preview)](https://github.com/asyncapi/vs-asyncapi-preview/issues/235) Plugin by adding [EDAVisualiser's](https://github.com/asyncapi/EDAVisualiser/) application focus view. This new feature will allow users to visualize how an application interacts with other defined applications. The project involves integrating the EDAVisualiser library, implementing a configuration system for related AsyncAPI documents, and creating a user interface to switch between preview modes. Skills required include JavaScript/TypeScript, VS Code extension development, and AsyncAPI knowledge.

> 9 | [Onboarding Contributor Guides](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10610947) | Docs | [V. Thulisile Sibanda](https://github.com/thulieblack, [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |The AsyncAPI documentation must include a persona-driven journey for different roles, such as documentation contributors, code contributors, ambassadors, maintainers, etc. The onboarding experience for new contributors isn't as efficient yet, and to solve this problem, we need to develop a series of onboarding guides to automate the onboarding process for new contributors.

Reviewer: ```suggestion
9 | [Onboarding Contributor Guides](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10610947) | Docs | [V. Thulisile Sibanda](https://github.com/thulieblack), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |The AsyncAPI documentation must include a persona-driven journey for different roles, such as documentation contributors, code contributors, ambassadors, maintainers, etc. The onboarding experience for new contributors isn't as efficient yet, and to solve this problem, we need to develop a series of onboarding guides to automate the onboarding process for new contributors.
```

> 9 | [Onboarding Contributor Guides](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10610947) | Docs | [Quetzalli Writes](https://github.com/quetzalliwrites), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |The AsyncAPI documentation must include a persona-driven journey for different roles, such as documentation contributors, code contributors, ambassadors, maintainers, etc. The onboarding experience for new contributors isn't as efficient yet, and to solve this problem, we need to develop a series of onboarding guides to automate the onboarding process for new contributors.

Reviewer: ```suggestion
9 | [Onboarding Contributor Guides](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10610947) | Docs | [V. Thulisile Sibanda](https://github.com/thulieblack, [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |The AsyncAPI documentation must include a persona-driven journey for different roles, such as documentation contributors, code contributors, ambassadors, maintainers, etc. The onboarding experience for new contributors isn't as efficient yet, and to solve this problem, we need to develop a series of onboarding guides to automate the onboarding process for new contributors.
```

> 9 | [Onboarding Contributor Guides](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10610947) | Docs | [Alejandra Quetzalli](https://github.com/quetzalliwrites), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |The AsyncAPI documentation must include a persona-driven journey for different roles, such as documentation contributors, code contributors, ambassadors, maintainers, etc. The onboarding experience for new contributors isn't as efficient yet, and to solve this problem, we need to develop a series of onboarding guides to automate the onboarding process for new contributors.

Reviewer: ```suggestion
9 | [Onboarding Contributor Guides](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10610947) | Docs | [Quetzalli Writes](https://github.com/quetzalliwrites), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |The AsyncAPI documentation must include a persona-driven journey for different roles, such as documentation contributors, code contributors, ambassadors, maintainers, etc. The onboarding experience for new contributors isn't as efficient yet, and to solve this problem, we need to develop a series of onboarding guides to automate the onboarding process for new contributors.
```

10 | [Implementation and integration of new layout for studio](https://github.com/asyncapi/studio/issues/634) | [AsyncAPI Studio](https://github.com/asyncapi/studio) | [Ashish Padhy](https://github.com/Shurtu-gal) & [Fran Méndez](https://github.com/fmvilas) |Studio goes beyond a simple playground to try out the spec and becomes a fully-featured suite to help developers work with event-driven architectures (EDA) and APIs. It will become a tool you can use to easily design your services and share them with your colleagues.

> 11 | [Path to Maintainership for the AsyncAPI React Project](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10811342) | Documentation | [Azeez Elegbede](https://github.com/AceTheCreator) |The [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react) is a handy tool for rendering documentation from your asynchronous APIs, all based on the AsyncAPI specification format. Plus, it helps validate that spec too.

Reviewer: ```suggestion
11 | [Path to Maintainership for the AsyncAPI React Project](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10811342) | Coding | [Azeez Elegbede](https://github.com/AceTheCreator) |The [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react) is a handy tool for rendering documentation from your asynchronous APIs, all based on the AsyncAPI specification format. Plus, it helps validate that spec too.
```

> 11 | [Path to Maintainership for the AsyncAPI React Project](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10811342) | Documentation | [Azeez Elegbede](https://github.com/AceTheCreator) |The [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react) is a handy tool for rendering documentation from your asynchronous APIs, all based on the AsyncAPI specification format. Plus, it helps validate that spec too.

Reviewer: Please update the project area

Reviewer 2: Thanks for the review. 

cc @SaxenaAnushka102

Reviewer: @SaxenaAnushka102, the project area for the AsyncAPI React project is incorrect; please update it

Reviewer: @SaxenaAnushka102 change the project area, so we can merge this

> 12 | [Expand Community Docs](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10818908) | Docs | [Quetzalli Writes](https://github.com/quetzalliwrites), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |Automating the AsyncAPI onboarding experience also involves documenting important workflows and processes for community members involved.

Reviewer: ```suggestion
12 | [Expand Community Docs](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10818908) | Docs | [V. Thulisile Sibanda](https://github.com/thulieblack), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |Automating the AsyncAPI onboarding experience also involves documenting important workflows and processes for community members involved.
```

> 12 | [Expand Community Docs](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10818908) | Docs | [Alejandra Quetzalli](https://github.com/quetzalliwrites), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |Automating the AsyncAPI onboarding experience also involves documenting important workflows and processes for community members involved.

Reviewer: ```suggestion
12 | [Expand Community Docs](https://github.com/orgs/asyncapi/discussions/1361#discussioncomment-10818908) | Docs | [Quetzalli Writes](https://github.com/quetzalliwrites), [Wisdom](https://github.com/wise4rmgod), [Anthony](https://github.com/bandantonio) |Automating the AsyncAPI onboarding experience also involves documenting important workflows and processes for community members involved.
```
